docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -786,4 +786,136 @@ spec:
   platforms: macOS
   tags: compliance, CIS, CIS1.4
   platform: darwin
+  contributors: GuillaumeRoss
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: Automatic security and data file updates is enabled (macOS)
+  query: SELECT 1 FROM managed_policies WHERE domain='com.apple.SoftwareUpdate' AND name='CriticalUpdateInstall' AND value=1 LIMIT 1;
+  description: "Checks that the operating system is configured to automatically download updates to built-in macOS security tools such as malware removal tools."
+  resolution: "Contact your IT administrator to ensure your Mac is receiving a profile that enables automatic security and data update installation."
```

Comment:
Is this another one where we are stuck querying managed policies rather than querying the underlying thing itself?

Reply:
In general, I think an end user would not understand the term "profile" in this context. I think the appropriate term should be "setting."